### DISCUSSION TITLE
libgstapp error on WSL 2 Ubuntu

### DISCUSSION THREAD

**Discussion by frankhuurman** [flet-dev/flet#627] [Q&A]
Hi everyone,

I just stumbled upon this project and can't wait to get going with it.
Though after installing flet and copying the counter example it can't seem to find a shared library:
`/.flet/bin/flet-0.1.65/flet/flet: error while loading shared libraries: libgstapp-1.0.so.0: cannot open shared object file: No such file or directory`

My initial thought is that maybe the Ubuntu 20.04 image on WSL2 is missing this lib by default?

I'm on Python 3.8.10, made a venv and pip installed flet in it.

If anyone has an idea, I'd greatly appreciate it! 😄

**Comment by FeodorFitsner** [comment]
Solved here: https://github.com/flet-dev/flet/issues/670
